ts/src/index.ts

```diff
@@ -192,33 +215,17 @@ export async function createProfiler(config: Config = {}): Promise<Profiler> {
  */
 export async function start(config: Config = {}): Promise<void> {
   let profiler: Profiler;
-  try {
-    profiler = await createProfiler(config);
-  } catch (e) {
-    logError(`${e}`, config);
-    return;
-  }
+  profiler = await createProfiler(config);
```

Comment:
No need to keep the declaration and assignment separate anymore. `const profiler = await createProfiler(config)`.

Reply:
Thanks! Done.